configuration/drugs/drugs.csv

```diff
@@ -123,8 +123,8 @@ e9399f39-10dd-4f01-901e-c6ab6fe714ad,,Glycopyrrolate 0.2mg/ml,9643b503-02ec-4fde
 60f0c3c0-ae59-4a37-8bda-1e9e5ec94e61,,Griseofulvin 500mg,c9c0c627-7ba6-4880-a702-cfc985a296fa,fdbe907b-6abc-41dc-9599-2d84105e4063,500mg,
 64e8f2e3-e70c-4184-9c73-178c806cf137,,Halperidol 5mg/ml,5831575c-89a8-45d5-a02e-ac194d70c064,bd1b2599-84d2-4f33-ba25-4f6d42d147a5,5mg/ml,
 dd4c9418-b342-45da-81ae-937d3774b14e,,Heparin 5000unit/ml,d2b1ec52-b6e8-4646-be5b-6c5a850233b4,bd1b2599-84d2-4f33-ba25-4f6d42d147a5,5000unit/ml,
-3b3fa759-b4c3-49e7-ab55-e78cb99af00c,,Hialorunate de Na+ 4mg/ml,b07006a3-62c1-485a-b8ef-af0de629d04c,7c46524c-a14f-43b6-b4b7-792cee31d8d5,4mg/ml,
-79b6d186-f048-456b-bd42-288140dfc6c9,,Hialorunate de Na+ et oxymetazoline,bd8f1042-b1bd-4bb2-8b3b-fbda065e371f,7c46524c-a14f-43b6-b4b7-792cee31d8d5,,
+3b3fa759-b4c3-49e7-ab55-e78cb99af00c,,Hialorunate de Sodium 4mg/ml,b07006a3-62c1-485a-b8ef-af0de629d04c,7c46524c-a14f-43b6-b4b7-792cee31d8d5,4mg/ml,
+79b6d186-f048-456b-bd42-288140dfc6c9,,Hialorunate de Sodium et oxymetazoline,bd8f1042-b1bd-4bb2-8b3b-fbda065e371f,7c46524c-a14f-43b6-b4b7-792cee31d8d5,,
```

Comment:
Let us make the 'Sodium' as 'sodium' seeing this is in fr

Reply:
Done